src/docs/product/sentry-basics/guides/alert-notifications/metric-alerts.mdx

```diff
@@ -1,20 +1,22 @@
 ---
 title: Metric Alerts
-sidebar_order: 1
+sidebar_order: 2
 description: "Learn more about creating and managing Metric Alerts."
 ---
 
-Create and manage Metric Alerts by navigating to Alerts and clicking the "Alert Rules" tab, where you'll see a list of all active rules and can add new rules or modify existing ones. Then, select your project and choose Metric Alert.
+Manage metric alerts by navigating to **Alerts** and clicking the "Alert Rules" tab. In this tab, you can see a list of all active rules and you can add new rules or modify existing ones. The minimum role required to create alerts is member. Sentry users with manager or owner permissions can change the minimum role requirement in **Settings > General Settings > Let Members Create and Edit Alerts**.
 
-The minimum role required to create alerts is member. Sentry users with manager or owner permissions can change the minimum role requirement in **Settings > General Settings > Let Members Create and Edit Alerts**.
+To create an alert, click "Create New Alert", choose your project, and then select what you want to be alerted about. All of the options except "Issues" create a metric alert.
 
-## 1. Choose Errors or Transactions
+## 1. Choose Errors or Performance
 
-In the Alert Conditions section, choose between [Errors](/platforms/javascript/usage/#capturing-errors) or [Transactions](/product/performance/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate).
+Select what you want to be alerted about from a number of preset alerts under "Errors" or "Performance". Your selection determines some of the alert properties, including whether you're alerting on [errors](/platforms/javascript/usage/#capturing-errors) or [transactions](/product/performance/). Errors provide data around a break, such as new issues or total errors in your project. Transactions provide data on the performance of your application, such as [latency](/product/performance/metrics/#latency) and [failure rate](/product/performance/metrics/#failure-rate). You can also choose to create a custom metric.
```

Comment:
general q: we use the phrase "alerting on", which focuses on what Sentry does. I think we might want to switch to "alerted to", which focuses on the result of the users's set up. Your call.